applications/callflow/src/cf_util.erl

```diff
@@ -339,7 +339,16 @@ vm_count_by_owner(<<_/binary>> = AccountDb, <<_/binary>> = OwnerId) ->
 %%--------------------------------------------------------------------
 -spec alpha_to_dialpad(ne_binary()) -> ne_binary().
 alpha_to_dialpad(Value) ->
-    << <<(dialpad_digit(C))>> || <<C>> <= strip_nonalpha(wh_util:to_lower_binary(Value))>>.
+    Stripped = [strip_nonalpha(C) || <<C>> <= wh_util:to_lower_binary(Value)],
```

Comment:
This all seems inefficient. When I look at it, I see three things happening:

1. Iterate through each character of `Value`
2. Filter out non a-z characters
3. Convert characters to dialpad digits.

This can be solved with a binary comprehension without the need to build intermediary lists and filter empty lists and all the ruckus below.

My suggestion:

```erlang
-spec alpha_to_dialpad(ne_binary()) -> ne_binary().                                           
alpha_to_dialpad(Value) ->                                                    
    << <<(dialpad_digit(C))>>                                                                                 
       || <<C>> <= wh_util:to_lower_binary(Value),
          is_alpha(C)                                                                        
    >>.                                                                         

-spec is_alpha(char()) -> boolean().
is_alpha(Char) ->      
    Char =< $z andalso Char >= $a.
```

IMO, this more clearly shows the three stages and doesn't build intermediate representations. It also passes all the tests!

Thoughts?